xblock/runtime.py

```diff
@@ -919,6 +934,18 @@ def _wrap_ele(self, block, view, frag, extra_data=None):
         wrapped.add_fragment_resources(frag)
         return wrapped
 
+    def _css_classes_for(self, block, view):
+        """
+        Get the list of CSS classes that the wrapping <div> should have for the
+        spcified xblock or aside's view.
```

Comment:
"specified"

Reply:
I really need to find a better spell checker plugin for my editor, or something... thank you :)